react-state/my-app/src/ToggleButton.js

```diff
@@ -0,0 +1,19 @@
+import { useState } from 'react';
+
+export default function ToggleButton({text, color}) {
+
+  const [isClicked, setIsClicked] = useState(false);
+  console.log('useState', isClicked);
+  function handleClick() {
+    console.log('useState', isClicked);
+    setIsClicked(!isClicked);
+    console.log('useState', isClicked);
+  }
+  if (!isClicked) {
```

Comment:
This works, but there is quite a bit of duplicated code. The only difference between the two is the color.

Try to figure out how to do this without the duplicated code.